scripts/deploy.sh

```
@@ -0,0 +1,76 @@
+# yq 설치 확인
+if ! command -v yq &> /dev/null; then
+    echo "ERROR: yq가 설치되어 있지 않습니다. https://github.com/mikefarah/yq 참고"
+    exit 1
+fi
+
+# 서브모듈 secret yml에서 DB 정보를 파싱하여 .env 생성
+parse_db_info() {
+    local file="$1"
+    local prefix="$2"
+
+    local username=$(yq '.spring.datasource.username' "$file")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)